app/services/iiif_print/manifest_builder_service_behavior_decorator.rb

```diff
@@ -2,7 +2,35 @@
 
 # OVERRIDE IiifPrint v1.0.0 to not render thumbnail files in the UV
 
+# rubocop:disable Metrics/BlockLength
 IiifPrint::ManifestBuilderServiceBehavior.module_eval do
+  def build_manifest(presenter:)
+    manifest = manifest_factory.new(presenter).to_h
+    hash = JSON.parse(manifest.to_json)
+    parent_and_child_solr_hits = parent_and_child_solr_hits(presenter) if @child_works.present?
+    hash = send("sanitize_v#{@version}", hash: hash, presenter: presenter, solr_doc_hits: parent_and_child_solr_hits)
+    if @child_works.present? && IiifPrint.config.sort_iiif_manifest_canvases_by
+      send("sort_canvases_v#{@version}",
+           hash: hash,
+           sort_field: IiifPrint.config.sort_iiif_manifest_canvases_by)
+    end
+    hash['rendering'] = rendering(presenter: presenter)
+    hash
+  end
+
+  def rendering(presenter:)
+    fsps = presenter.file_set_presenters.reject { |fsp| fsp.mime_type.include?('image') }
```

Comment:
glad ya'll got this figured out! well done! 

nit picking here 😓 I personally prefer more descriptive/semantic variable names even if they are longer.

Reply:
good point! i can make the change